README.md

```diff
@@ -0,0 +1,128 @@
+
+Now, navigate to the `view` directory within the codebase and run the following command (The command below assumes that the name of the directory you cloned into is `farmlink`. If it is not, replace farmlink in all paths with the name you gave the directory):
+
+    ~/farmlink/view npm install
+
+Now repeat the process in the functions directory:
+
+    ~/farmlink/functions npm install
+
+Now you can test if the React application runs by running the following command in the view directory (if it doesn't run, you will likely need to download a missing dependency)
+
+    ~/farmlink/functions npm start
+
+If it runs, then you are good to go onto the next step!
+
+## Step #3: Setup Firebase
+
+Create a project in Firebase (go to this [tutorial](https://firebase.google.com/docs/projects/learn-more) to learn how) and run the following after doing so:
+
+    ~/farmlink/functions npm install -g firebase-tools
+    firebase init
+
+Once you have run `firebase init`, answer the prompts as follows:
+
+    Which Firebase CLI features do you want to set up for this folder? 
+    => Functions: Configure and deploy Cloud Functions
+    First, let’s associate this project directory with a Firebase project...
+    => Use an existing project
+    Select a default Firebase project for this directory
+    => application_name 
+    Do you want to use ESLint to catch probable bugs and enforce style?
+    => No
+    Do you want to install dependencies with npm now?
+    => Yes
+
+After this, you should see the following message:
+
+    ✔ Firebase initialization complete!
+
+## Step #3: Setup Firebase Firestore
+
+Go to the database tab in Firebase console and create a database set to `test mode` and choose your hosting location.
+
+If you would like to test Firebase Firestore locally, run `firebase serve` in the functions directory. We recommend using [Postman](https://www.postman.com/) as a tool for testing the APIs.
+
+If you get an error regarding credentials when you run `firebase serve`, then use the following steps to fix it:
+
+1. Go to the **Project Settings** tab in the Firebase Console
+2. Go to the **Service Accounts tab**
+3. There will be an option of **Generating a New Key**. Click on that option and it will download a file with a `.json` extension.
+4. Use the command below to connect these credentials 
+
+    export GOOGLE_APPLICATION_CREDENTIALS="/home/user/Downloads/[FILE_NAME].json
+
+And run: `firebase login --reauth`
+
+## Step #4: Setup Firebase Authentication
+
+Go to the **Project Settings > General** in Firebase Console and select the web application icon (which should contain `</>` in the icon). Now follow the process for making the web application. At the end of the process you will click **Continue to Console** and see a `.json` with the firebase config. Paste the contents of this `.json` into the `config.js` file in the **functions > util** directory of the application repository.
```

Comment:
Not totally necessary, but for these UI-heavy sections a screenshot or two would be very helpful!

Reply:
where would I put the image files in the repository or should I create public links to them?

Reply:
Kind of anywhere you want.  You'll probably want them separate from the source code so people don't get confused.  [Here's some advice](https://stackoverflow.com/questions/10189356/how-to-add-screenshot-to-readmes-in-github-repository) on how to link them.  There's a clever idea there that will allow you to avoid checking them into master at all:

> Additionally, if you create a new branch screenshots to store the images you can avoid them being in the master working tree

Reply:
So that Angeline can update her README, I am going to merge this branch. I will add some pictures or gifs then create a different pull request! Definitely a great tip!